Lib/test/test_dbm_gnu.py

```diff
@@ -205,6 +207,11 @@ def test_clear(self):
                 self.assertNotIn(k, db)
             self.assertEqual(len(db), 0)
 
+    @support.run_with_locales('LC_ALL', 'fr_FR.UTF-8', 'fr_FR.iso88591')
+    def test_localized_error(self):
+        expect = re.escape('Base de données vide')
+        empty = create_empty_file(filename)
+        self.assertRaisesRegex(gdbm.error, expect, gdbm.open, filename, 'r')
```

Comment:
Unfortunately, non-existing ASCII name wouldn't work because the error would be an errno error instead of a `gdbm.error`. I really need a specific kind of error to trigger in order to hit the translated messages. But I can't create an empty file with UTF8-undecodable byte names I think (os.open() fails with a ValueError). So I'm not entirely sure how I can do it =/ I'll try to find other ways offline (I will be offline until Friday)

Reply:
How come `create_empty_file` failed when I tried? I'll need to check it later. But thanks! (I remember trying using the FN_NONDECODABLE file but didn't try with just '\xff').